runtime/runtime/src/config.rs

```diff
@@ -162,7 +189,7 @@ pub fn total_deposit(actions: &[Action]) -> Result<Balance, BalanceOverflowError
 }
 
 /// Get the total sum of prepaid gas for given actions.
-pub fn total_prepaid_gas(actions: &[Action]) -> Result<Gas, GasOverflowError> {
+pub fn total_prepaid_gas(actions: &[Action]) -> Result<Gas, IntegerOverflowError> {
     let mut total_gas: Gas = 0;
```

Comment:
Folks at security review suggested to reimplement this function with try_fold to avoid `mut`.

Reply:
Fixed. Would be interesting to know though the reasoning to avoid mut here..